### DISCUSSION TITLE
Deploy longhorn in /opt/longhorn

### DISCUSSION THREAD

**Discussion by athulg93** [longhorn/longhorn#9134] [Q&A]
Hey,
Bydefault longhorn deploys in /var/lib.
It also creates some folders in /var

How do i configure this to ensure that everything is created in /opt instead?
I opened up the config.yaml file and switched out the locations from /var to /opt and longhorn was deployed but the **share-manager-pvc** pods fail to deploy. It starts and immediately goes into **Terminating** stage

When i do **kubectl -n longhorn-system get event** I can see the following lines being repeated multiple times
17m         Normal    Created                  pod/share-manager-pvc-4f9b89e6-3064-4142-8741-039e8386abed    Created container share-manager
17m         Normal    Started                  pod/share-manager-pvc-4f9b89e6-3064-4142-8741-039e8386abed    Started container share-manager
17m         Normal    Killing                  pod/share-manager-pvc-4f9b89e6-3064-4142-8741-039e8386abed    Stopping container share-manager
17m         Warning   Unhealthy                pod/share-manager-pvc-4f9b89e6-3064-4142-8741-039e8386abed    Readiness probe failed: cat: /var/run/ganesha.pid: No such file or directory
16m         Normal    Scheduled                pod/share-manager-pvc-4f9b89e6-3064-4142-8741-039e8386abed    Successfully assigned longhorn-system/share-manager-pvc-4f9b89e6-3064-4142-8741-039e8386abed to node1.lab.archivas.com

Any idea how to handle this?

**Comment by ChanYiLin** [comment]
I don't think this is related to the default path, it seems the pod failed to access the pid file on the host.
Hi @derekbit @c3y1huang Do you have any clue?

**Comment by c3y1huang** [comment]
@athulg93 Could you check the logs for the share-manager pod? Also, have you installed the NFSv4 client on the cluster nodes?

https://longhorn.io/docs/1.6.2/nodes-and-volumes/volumes/rwx-volumes/#requirements
